src/components/Adminviewuser.js

```diff
@@ -0,0 +1,94 @@
+import React, {useState} from 'react';
+import ViewUser from './Viewuser';
+import Popup from 'reactjs-popup';
+import './Admin.scss';
+
+const LookupUser = (props) => {
+
+	const [user, setUser] = useState('');
+	const [userData, setUserData] = useState(null);
+	const [isOpen, setIsOpen] = useState(false);
+
+	const fetchUser = (user) => {
+		setUserData(null);
+		let headers = {
+			'Authorization': 'Bearer ' + localStorage.getItem('token')

```

Comment:
(in future we can factor this auth header out into a function like we have in the whats-cookin repo, but this is fine for now!)

Reply:
added new commit to address this